components/common/WalletModal.vue

```diff
@@ -125,7 +126,7 @@ export default class WalletModal extends Vue {
     }
   }
 
-  protected setWallet(wallet: Wallet): void {
+  protected async setWallet(wallet: BaseDotsamaWallet): Promise<void> {
```

Comment:
you use async here but there is no await in method
why did you change the type? in future we want to authorize not only dotsama wallet

Reply:
you are right. I will remove it